jbmc/src/java_bytecode/java_bytecode_parse_tree.h

```diff
@@ -84,6 +84,8 @@ class java_bytecode_parse_treet
       is_private(false), is_static(false), is_final(false)
     {
     }
+
+    virtual ~membert() = default;
```

Comment:
make final?

Reply:
But ... this class has children deriving from it?

Reply:
Well, there is case to be made for private inheritance, the use of `virtual` indeed does not seem to be justified.